src/salesforce/content-mapper.ts

```diff
@@ -7,16 +7,29 @@ import { SalesforceArticleLayoutItem } from './model/salesforce-article-layout-i
 import { GeneratedValue } from '../utils/generated-value.js';
 import { LabelReference } from '../model/label-reference.js';
 import { ExternalLink } from '../model/external-link.js';
+import { SalesforceConfig } from './model/salesforce-config';
 
 const EXCLUDED_FIELD_TYPES = ['DATE_TIME', 'LOOKUP', 'CHECKBOX'];
 
 export function contentMapper(
   categoryGroups: SalesforceCategoryGroup[],
   articles: SalesforceArticleDetails[],
-  salesforceArticleContentFields: string[],
+  config: SalesforceConfig,
   fetchCategories: boolean,
+  buildExternalUrls: boolean,
 ): ExternalContent {
   const labelsMapping = buildIdAndNameMapping(categoryGroups);
+  const contentFields = (
+    config.salesforceArticleContentFields?.split(',') || []
+  )
+    .map((f) => f.trim())
+    .filter((f) => f.length > 0);
+  const baseUrl = config.salesforceLightningBaseUrl;
+  const lightningLanguageCode = config.salesforceLanguageCode
+    ? config.salesforceLanguageCode
```

Comment:
shouldn't the salesforce LANGUAGE_MAPPING be applied here, too?
What are the expected values for config.salesforceLanguageCode? genesys knowledge language codes? For example if config.salesforceLanguageCode is "de-DE", what is the expected url?